blocks/components/editable/index.js

```diff
@@ -112,11 +112,18 @@ export default class Editable extends wp.element.Component {
 	}
 
 	getRelativePosition( node ) {
-		const editorPosition = this.editor.getBody().closest( '.editor-visual-editor__block' ).getBoundingClientRect();
 		const position = node.getBoundingClientRect();
+
+		// Find the parent "relative" positioned container
+		const container = this.props.inlineToolbar
+			? this.editor.getBody().closest( '.blocks-editable' )
+			: this.editor.getBody().closest( '.editor-visual-editor__block' );
+		const editorPosition = container.getBoundingClientRect();
+
+		// The margins are different depending on the container
 		return {
-			top: position.top - editorPosition.top + 40 + ( position.height ),
-			left: position.left - editorPosition.left - 157
+			top: position.top - editorPosition.top + position.height + ( this.props.inlineToolbar ? 50 : 40 ),
+			left: position.left - editorPosition.left - ( this.props.inlineToolbar ? 100 : 157 )
```

Comment:
Yes, the `40` and `50` are just the top values of the toolbar component (depending if it's inline or not). This is not perfect right! but it does the job right now. We can improve it later :)